release/lib/git-helpers.ts

```diff
@@ -89,21 +95,19 @@ const getResponseForCommitInfoRequest = async (commitSHA: string) => {
          * which in most cases, is wrongly set.
          */
 
-        const cachedAuthor = commitInfo.author && authors.get(commitInfo.author.login);
+        const cachedAuthor = authors.get(commitInfo.author.login);
 
         if (cachedAuthor) {
-            debug(`Reusing cached Author ${commitInfo.author?.login}`);
+            debug(`Reusing cached Author ${commitInfo.author.login}`);
 
             return cachedAuthor;
         }
 
-        debug(`Requesting info for login ${commitInfo.author?.login}`);
+        debug(`Requesting info for login ${commitInfo.author.login}`);
 
         const responseForUserInfoRequestPromise = getResponseForUserInfoRequest(commitInfo);
```

Comment:
Note: This call specifically didn't expect author to be `null`, we just hadn't hit it in a previous release. The GitHub libraries were updated to allow `null` ~8 months ago. I updated other cases flagged by TypeScript then but missed this one. Fixed by making the `null` handling an early return instead.